native/explorer/Cargo.toml

```diff
@@ -79,7 +81,7 @@ version = "0.31"
 [features]
 default = ["ndjson", "cloud", "nif_version_2_15"]
 
-cloud = ["aws"]
+cloud = ["object_store", "tokio", "tokio-util", "aws"]
```

Comment:
Which way around do we want these features to depend?

Do we want the cloud feature to depend on all possible provider features (e.g. aws/gcp/azure/...), or have cloud be the 'base' feature and features like aws depend on it?

Polars itself does the former.

Reply:
I don't think we expect anyone to ever use them. It is only used internally to exclude those projects in certain targets.

Reply:
Yeah, it's only used to restrict some features to some targets (mostly, RISCV and ARM based targets).

But, since you mentioned, I think a better way to organize would be to have `cloud` as a base, and we have `aws` as the feature we enable / disable. Perhaps we could only have the `aws` feature for now - without `cloud`-, since we don't have any other cloud provider.  WDYT?

Reply:
I think I won't change this for now. Going to postpone a little bit.